### PR TITLE
(@bem-react/core): complete readme

### PR DESCRIPTION
**Pull request by Vittly**

### CODE REVIEW

**Comment by Vittly** [comment]
🔥

File: packages/core/README.md [hunk 1]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+import * as React from 'react';
+import { IButtonProps } from './index';
+
+export const Button: React.SFC<IButtonProps> = ({ text, className }) => (
+    <div className={className}>{text}</div>
+);
 
-const Presenter: React.SFC<IPresenterProps> = ({ className }) =>
-    <div className={presenter({}, [className])} />;
+// Button looking as a link
```
Comment (krishnevsky) [inline packages/core/README.md]: Button that looks like a link

File: packages/core/README.md [hunk 2]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+import * as React from 'react';
+import { withBemMod, ModBody } from '@bem-react/core';
+import { IButtonProps } from './index';
+
+const ButtonLink: ModBody<IButtonProps> = (Base, { text, className }) => (
+    // className === 'Button Button_type_link'
+    <a className={className}>{text}</a>
+);
+
+// should be read like:
```
Comment (krishnevsky) [inline packages/core/README.md]: capitalize S

File: packages/core/README.md [hunk 3]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+//   else → return Base
+export const ButtonTypeLink = withBemMod<IButtonProps>('Button', { type: 'link' }, ButtonLink);
+
+// Button for actions (has different styles)
+// components/Button/_theme/Button_theme_action.tsx
+
+import { withBemMod } from '@bem-react/core';
+import { IButtonProps } from './index';
+
+// should be read like:
```
Comment (krishnevsky) [inline packages/core/README.md]: capitalize S

File: packages/core/README.md [hunk 4]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
 ```
 
-### compose
+### Debug
+
+To help your debug "@bem-react/core" support development mode.
 
-``` js
-import { compose } from '@bem-react/core';
+For `<Button text="Hello" type="link" theme="action" />` (from **Example** above) React DevTools will show:
```
Comment (krishnevsky) [inline packages/core/README.md]: ... (from the **Example** above), React DevTools will show:

File: packages/core/README.md (hunk 1 above)
Comment (krishnevsky) [inline packages/core/README.md]: Button that looks like a link

File: packages/core/README.md [hunk 5]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+import { compose, IClassNameProps } from '@bem-react/core';
+
+import { Button as Base } from './Button';
+import { ButtonTypeLink } from './_type/Button_type_link';
+import { ButtonThemeAction } from './_theme/Button_theme_action';
+
+export interface IButtonProps extends IClassNameProps {
+    text: string;
+
+    // list of all modifiers
```
Comment (krishnevsky) [inline packages/core/README.md]: capital L

File: packages/core/README.md [hunk 6]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+    type?: 'link';
+    theme?: 'action';
 }
 
-const presenter = cn('Presenter');
+// composition of all variations
```
Comment (krishnevsky) [inline packages/core/README.md]: Combination of all variations 
(?)

File: packages/core/README.md [hunk 7]
```diff
@@ -6,33 +6,84 @@ Tiny helper for [BEM modifiers](https://en.bem.info/methodology/key-concepts/#mo
...
+// JSX → <Button text="Hello" type="link" theme="action" /> gives
+// HTML → <a class="Button Button_type_link Button_theme_action">Hello</a>
+export const Button = compose(
+    ButtonThemeAction,
+    ButtonTypeLink
+)(Base);
+
+// Button as is
```
Comment (krishnevsky) [inline packages/core/README.md]: Normal button (?)
Button without modifications (?)

**Pull request merged**